src/components/Content/ContentList.js

```diff
@@ -16,10 +17,8 @@ class ContentListElement extends React.Component {
     return (
       <Link href={this.props.url}>
         <div className={style.row}>
-          <div className={cx(style.elementImage, style.element)}>
-            <img className={style.image} src={this.props.page.thumbnail}/>
-          </div>
-          <div className={style.elementText}>{this.props.page.title}</div>
+          <img className={style.image} src={this.props.page.thumbnail ? this.props.page.thumbnail : IconPlaceholder}/>
```

Comment:
Can be simplified to `<img className={style.image} src={this.props.page.thumbnail || IconPlaceholder}/>`